packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -1,7 +1,13 @@
 <template>
   <template if:true={shouldDisplay}>
-    <li class={tabClass} title={label} role="presentation">
-      <a class="slds-tabs_default__link" role="tab" onclick={select}>{label}</a>
-    </li>
+    <button
+      class={tabClass}
+      onclick={select}
+      aria-label={arialLabelText}
+      aria-pressed={isPressed}
```

Comment:
Ah well you can just use this then:

```suggestion
      aria-pressed={isActive}
```

Reply:
comments added for isPressed